taskcat/_config.py

```diff
@@ -208,18 +207,18 @@ def _assign_account_buckets(self):
                     bucket_dict,
                     config=self,
                     name=self.s3bucket.name,
-                    client=test_region.client,
+                    client=test_region.client("s3"),
```

Comment:
This call return a session correct ? `test_region.client("s3")  `
So this would make it a bit clearer
session = `test_region.client("s3")`

Reply:
`s3bucket.client` (should) contain a boto3 client (equivalent of `boto3.client('s3')`), not session, wanted to make it an actual s3 client so that the consumer is not left with the task of ensuring the correct region.

Reply:
@avattathil - Sorry, Jay is right. My bad for leading you in the wrong direction.